packages/query-codemods/eslint.config.js

```diff
@@ -9,6 +8,7 @@ export default [
     rules: {
       'cspell/spellchecker': 'off',
       '@typescript-eslint/no-unnecessary-condition': 'off',
+      '@typescript-eslint/require-await': 'off',
```

Comment:
@minseong0324  I'm going to treat it like this now. 

We can apply @typescript-eslint/require-await rule as error to the rest by turning off the this rule only for @tanstack/query-codemods